rsmanage/rsmanage.py

```diff
@@ -280,21 +280,19 @@ def shutdown(config):
 @click.option(
     "--start-date", default="2001-01-01", help="Start Date for the course in YYYY-MM-DD"
 )
-@click.option("--python3", is_flag=True, default=True, help="Use python3 style syntax")
```

Comment:
I'm Not sure what the difference is between old and new, I thought that is_flag automatically implied the --no version of the option.  But maybe this is more clear/explicit.

Reply:
Per the [click docs](https://click.palletsprojects.com/en/8.0.x/options/#boolean-flags), the old code creates an option without the `--no` version of the option:

> If you really don’t want an off-switch, you can just define one and manually inform Click that something is a flag:

```
import sys

@click.command()
@click.option('--shout', is_flag=True)
```